node/src/validator/mod.rs

```diff
@@ -158,22 +177,47 @@ impl<N: Network, C: ConsensusStorage<N>> Validator<N, C> {
             fs::copy(format!("{bft_path}/../.parameters.json"), format!("{bft_path}/.parameters.json")).unwrap();
         }
 
+        let base_path =
```

Comment:
this is the same path as at L153-154; we can reuse the first one

Reply:
I don't think this works without the trailing `/`, unfortunately—I'll check again though.